Site/SiteConcentrateFileFinder.php

```diff
@@ -10,34 +10,25 @@ public function getDataFiles()
     {
         // Load data files from composer module directories and from site
         // dependency directory.
-        return array_merge(
-            $this->getSiteDataFiles(),
-            $this->getComposerDataFiles()
-        );
+        return [
+            ...$this->getSiteDataFiles(),
+            ...$this->getComposerDataFiles(),
+        ];
     }
 
-    protected function getWwwPath()
+    protected function getRootPath(): string
     {
-        $www_path = realpath('.');
-
-        while (basename($www_path) !== 'www') {
-            $www_path = dirname($www_path);
-        }
-
-        return $www_path;
+        return dirname(__DIR__, 3);
```

Comment:
`getRootPath` now assumes a composer package structure instead of getting the location of the `index.php` script and navigating upward until `www` is located. This means it will work for scripts that do not live underneath the `www` folder, like cron jobs under `system`.